### DISCUSSION TITLE
Create Glossary

### DISCUSSION THREAD

**Discussion by GeorgiaHCA** [alan-turing-institute/AutSPACEs#378] [Ideas]
### Summary

In order to make the platform more inclusive, it would help to include a glossary of technical termss, as well as a gloassary of appropriate, inclusive language relating to autism and intersectional issues. 

Creating glossary here: https://hackmd.io/atchh4nOSCaRSbBHhd4wjA

### Deadline 

15th Feb 2021

### What needs to be done?

- [ ] List words that are technical and which may be confusing to non-technical people throughout repository and website
- [ ] List exclusionary language and appropriate language 
- [ ] Either compile into gloassry in markdown and add to appropriate place in the repository, or add glosssary of terms to each page. 

### Who can help?

Anyone can help by:
* noting words you do not know or find confusing
* suggesting good examples of inclusionary language or flagging inappropriate ones 
* making the glossary

This was requsested by community-members for both the rpository and the platform itself.

**Comment by Angelsaur** [comment]
Hi I just had a read through the website and have made a list of technical terms which newbies like myself might need help with. Most of them are already explained but it might be good to have all the explanations in one place so it’s easy to find 

Backend database 
Branch
Code of conduct
Dashboard
Django
HTML
Open source
Participatory science
Pull request
README
Repository 

I also think it might be helpful to explain the various tabs/ clickable things.

issues 
Actions
Pull requests
Projects 
Security
Insights 
Code

And then 
Releases
Packages 
Contributors
Environments 


Happy to make the glossary if someone more experienced can check it over/ help me if I get stuck.

Angelsaur

**Comment by lottycoupat** [comment]
@GeorgiaHCA think this should also be in the other repo?
